llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -952,6 +957,36 @@ class StoreInst final : public Instruction {
 #endif
 };
 
+class UnreachableInst final : public Instruction {
+  /// Use UnreachableInst::create() instead of calling the constructor.
+  UnreachableInst(llvm::UnreachableInst *I, Context &Ctx)
+      : Instruction(ClassID::Unreachable, Opcode::Unreachable, I, Ctx) {}
+  friend Context;
+  Use getOperandUseInternal(unsigned OpIdx, bool Verify) const final {
+    return getOperandUseDefault(OpIdx, Verify);
+  }
+  SmallVector<llvm::Instruction *, 1> getLLVMInstrs() const final {
+    return {cast<llvm::Instruction>(Val)};
+  }
+
+public:
+  static UnreachableInst *create(Instruction *InsertBefore, Context &Ctx);
+  static UnreachableInst *create(BasicBlock *InsertAtEnd, Context &Ctx);
+  static bool classof(const Value *From);
+  unsigned getNumSuccessors() const { return 0; }
+  unsigned getUseOperandNo(const Use &Use) const final {
+    return getUseOperandNoDefault(Use);
```

Comment:
This should have no operands, so I think it should be safe to replace `return getUseOperandNoDefault(Use);` with `llvm_unreachable("UnreachableInst has no operands!");`

Reply:
Now that I'm looking at it, do really need that function? I think it'd be safe to just remove it considering that `unreachable` has no operands.

Reply:
You need it because it is declared as `getUseOperandNo() = 0`. You will get a build error if you remove it.

Reply:
Oops, thanks for the correction. I'll make sure fix it.

Reply:
Done.